scale/scheduler/tasks/system_task.py

```diff
@@ -46,3 +51,18 @@ def _add_database_docker_params(self):
                      DockerParameter('env', 'SCALE_DB_PORT=%s' % db['PORT'])]
 
         self._docker_params.extend(db_params)
+
+    def _add_messaging_docker_params(self):
```

Comment:
Didn't see any unit test coverage for this function.

Reply:
This method is tested within the unit tests for the messaging service. Being a very light method, I don't think a separate test for it is necessary.